docs/sql-migration-guide.md

```diff
@@ -56,6 +56,8 @@ license: |
   In Spark 3.1 and earlier, table refreshing leaves dependents uncached.
 
   - In Spark 3.2, the usage of `count(tblName.*)` is blocked to avoid producing ambiguous results. Because `count(*)` and `count(tblName.*)` will output differently if there is any null values. To restore the behavior before Spark 3.2, you can set `spark.sql.legacy.allowStarWithSingleTableIdentifierInCount` to `true`.
+ 
+  - In Spark 3.2, a `SHOW TBLPROPERTIES` clause's output shows `key` and `value` columns whether you specify the property key. In Spark 3.1 and earlier, a `SHOW TBLPROPERTIES` clause's output only show a `value` column when you specify the property key.
```

Comment:
nit: "whether you specify the property key." => "whether you specify the table property \`key\`."

Reply:
> nit: "whether you specify the property key." => "whether you specify the table property `key`."

Done